inst/schema/report_run_request.json

```
@@ -2,7 +2,7 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "type": "object",
     "properties": {
-        "url": {
+        "ssh_key": {
             "type": "string"
```

```suggestion
            "type": ["string", "null"],
```

This can make the implementation of the client in Packit simpler (and presumably requires no code change to handle here).